specification/applicationinsights/resource-manager/Microsoft.Insights/stable/2015-05-01/componentWorkItemConfigs_API.json

```diff
@@ -173,15 +173,91 @@
         "responses": {
           "200": {
             "description": "The work item configuration that was successfully deleted.",
-            "schema": {              
-			  }
+            "schema": { }
           }
         },
         "x-ms-examples": {
           "WorkItemConfigurationDelete": {
             "$ref": "./examples/WorkItemConfigDelete.json"
           }
         }
+      },
+      "get": {
+        "description": "Gets specified work item configuration for an Application Insights component.",
+        "operationId": "WorkItemConfigurations_GetItem",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ResourceGroupNameParameter"
```

Comment:
Are you trying to reference a definition from another RP here. That is not recommended since you will be taking a dependency on some other service's definition (even though resourcegroupparameter is fairly consistent across all RPs)
tl;dr just reference these parameters in this spec itself or create a common definition spec in this directory

Reply:
Agreed. I'll talk to our leads on this one as this specific pattern is all over this service definition. Maybe we could address it in a separate PR?

Reply:
@debugthings I take this back. Looks like `common-types` is a common definition schema for all RPs and this is the way to go forward. So this does look correct. Keep this unchanged. Apologize for the confusion 😄